lib/engine/game/g_1856/game.rb

```diff
@@ -1600,13 +1600,32 @@ def national_token_swap
           # Other tokens second, ignoring duplicates from the home token set
           @nationalized_corps.each do |corp|
             corp.tokens.each do |token|
-              next if !token.used || !token.city || home_bases.include?(token.city.hex)
+              next if !token.used || !token.city || home_bases.map(&:hex).include?(token.city.hex)
```

Comment:
on here and 1638, probably more efficient to do

home_bases.any? { |base| base.hex == token.city.hex }

Reply:
Changed